api-platform/config/packages/doctrine.yaml

```diff
@@ -9,12 +9,12 @@ doctrine:
     auto_generate_proxy_classes: true
     naming_strategy: doctrine.orm.naming_strategy.underscore_number_aware
     auto_mapping: true
-    # mappings:
-    #   App:
-    #     is_bundle: false
-    #     dir: '/src/api-platform/src/Entity'
-    #     prefix: 'App\Entity'
-    #     alias: App
+    mappings:
+      App:
+        is_bundle: false
+        dir: '%kernel.project_dir%/src'
+        prefix: 'App\ApiResource'
+        alias: App
```

Comment:
I don't think we need this as I added the ResourceNameCollection service

Reply:
Isn't this service only related to api-platform resources, not doctrine mapping ?